frb_example/pure_dart/dart/lib/bridge_generated.dart

```diff
@@ -1347,6 +1348,70 @@ class FlutterRustBridgeExampleSingleBlockTestImpl implements FlutterRustBridgeEx
         argNames: ["id"],
       );
 
+  String syncDartOpaque({required Object notTemp, dynamic hint}) {
```

Comment:
btw, curious why is it named like "notTemp"

Reply:
forgot to change